frontend/public/components/modals/error-modal.jsx

```diff
@@ -2,14 +2,13 @@ import * as React from 'react';
 
 import {createModalLauncher, ModalTitle, ModalBody, ModalFooter} from '../factory/modal';
 
-export const errorModal = createModalLauncher(
-  ({error, cancel}) => {
-    return (
-      <div className="modal-content">
-        <ModalTitle>Error</ModalTitle>
-        <ModalBody>{error}</ModalBody>
-        <ModalFooter inProgress={false} errorMessage=""><button type="button" onClick={(e) => cancel(e)} className="btn btn-default">OK</button></ModalFooter>
-      </div>
-    );
-  }
-);
+export const ModalErrorContent = ({ error, title = 'Error', cancel }) => (
+  <div className="modal-content">
+    <ModalTitle>{title}</ModalTitle>
+    <ModalBody>{error}</ModalBody>
+    <ModalFooter inProgress={false} errorMessage="">
+      <button type="button" onClick={(e) => cancel(e)} className="btn btn-default">OK</button>
+    </ModalFooter>
+  </div>);
+
+export const errorModal = createModalLauncher(props => <ModalErrorContent {...props} />);
```

Comment:
Could this be just `createModalLauncher(ModalErrorContent)`?

Reply:
done